std-bits/table/src/main/java/org/enso/table/data/column/operation/RequiresNumberFormatting.java

```diff
@@ -0,0 +1,44 @@
+package org.enso.table.data.column.operation;
+
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.MixedStorage;
+import org.enso.table.data.table.Column;
+import org.graalvm.polyglot.Context;
+
+public class RequiresNumberFormatting {
+  /** Counts the number of cells in the columns with non trivial whitespace */
```

Comment:
Again, in this file it seems that comments were copied over from old file and not updated.